gravitee-apim-console-webui/src/portal/customization/top-bar/portal-top-bar.component.html

```diff
@@ -0,0 +1,116 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<portal-header title="Top Bar" subtitle="Customize the Developer Portal top bar." />
```

Comment:
It raised a flag with me seeing the `portal-header` in multiple places with the same text. To help with debugging/changing text in the future, it may be best to have a parent component that has the `portal-header` for Top Bar + `<router-outlet></router-outlet>` and then in the `routes` file, you have the table and edit components as children.